Applications/Tizen_native/CustomShortcut/src/view.c

```diff
@@ -27,16 +33,93 @@ int view_create(widget_context_h context, int w, int h) {
   evas_object_resize(win, w, h);
   evas_object_show(win);
 
+  // Adding conformant
   conform = elm_conformant_add(win);
+  if (conform == NULL){
+    dlog_print(DLOG_ERROR, LOG_TAG, "failed to create conformant object");
+    evas_object_del(win);
+    return WIDGET_ERROR_FAULT;
+  }
 
   evas_object_size_hint_weight_set(conform, EVAS_HINT_EXPAND, EVAS_HINT_EXPAND);
   elm_win_resize_object_add(win, conform);
   evas_object_show(conform);
 
+  // Adding naviframe
+  nf = elm_naviframe_add(conform);
+  if(nf == NULL){
+    dlog_print(DLOG_ERROR, LOG_TAG, "failed to create naviframe object");
+    evas_object_del(win);
+    return WIDGET_ERROR_FAULT;
+  }
+
+  elm_object_part_content_set(conform, "elm.swallow.content", nf);
+  eext_object_event_callback_add(nf, EEXT_CALLBACK_BACK, eext_naviframe_back_cb, NULL);
+  eext_object_event_callback_add(nf, EEXT_CALLBACK_MORE, eext_naviframe_more_cb, NULL);
+
+  evas_object_show(nf);
+
   widget_app_context_get_tag(context, (void **)&wid);
 
+  wid->circle_nf = eext_circle_surface_naviframe_add(nf);
+
+  wid->naviframe = nf;
   wid->win = win;
   wid->conform = conform;
 
   return status;
 }
+
+/**
+ * @brief creates layout from edj
+ * @param[in] context context of widget instance
+ */
+int view_create(widget_context_h context) {
+  widget_instance_data_s *wid = NULL;
+  int status = WIDGET_ERROR_NONE;
+
+  widget_app_context_get_tag(context, (void**) &wid);
+  wid->layout = _create_layout(wid->naviframe, wid->edj_path, "main", NULL, NULL);
+
+  if (wid->layout == NULL) {
+		dlog_print(DLOG_ERROR, LOG_TAG, "failed to create a layout of no alarm.");
+		evas_object_del(wid->win);
+		return WIDGET_ERROR_FAULT;
+	}
```

Comment:
need indent fix

Reply:
Oops, I should run clang format before push. Thanks :)

Reply:
Oops, I should run clang format before push. Thanks :)